app/views/shared/_csv_import_modal.html.erb

```diff
@@ -25,9 +25,10 @@
                 <li>
                   <%= f.file_field :file, accept: 'text/csv', class: 'form-control-file', style: "margin: auto;" %>
                 </li>
-                <li style="margin-bottom: 27px;">Then click the "Import CSV" button to import your <%= import_type.downcase %>.</li>                  <%= button_tag class: "btn btn-md btn-info" do %>
-                    <i class="fa fa-upload"></i> Import CSV
-                  <% end %>
+                <li style="margin-bottom: 27px;">Then click the "Import CSV" button to import your <%= import_type.downcase %>.</li>
+                <%= button_tag class: "btn btn-md btn-info", data: { disable_with: "Importing" } do %>
```

Comment:
Only thing changed was adding `data: { disable_with: "Importing" }`, otherwise rearranging the button tag to be in its own line.

Looked into writing a **system spec** for this but it's [near impossible to trigger a double click the way we want ](https://stackoverflow.com/questions/44191271/how-can-i-simulate-two-fast-clicks-on-a-submit-button-in-capybara). The various double click methods (`.click.click`, `double_click`, or using js `trigger`) don't actually let you double click.

Reply:
I think that's fine - if we are validating this on the backend, that's the most important check. Frontend is more of a nice to have.